graphics/lvgl/Makefile

```diff
@@ -25,32 +25,67 @@ include $(APPDIR)/Make.defs
 LVGL_DIR = .
 LVGL_DIR_NAME = lvgl
 
-# Relax format check for LVGL to avoid errors on prinf() use
+# Relax LVGL's format checking and unused variable checking to avoid errors
 
-CFLAGS += -Wno-format
+CFLAGS += -Wno-format -Wno-unused-variable
 
-# LVGL Libraries
+-include ./lvgl/lvgl.mk
 
--include ./lvgl/src/lv_core/lv_core.mk
--include ./lvgl/src/lv_hal/lv_hal.mk
--include ./lvgl/src/lv_widgets/lv_widgets.mk
--include ./lvgl/src/lv_font/lv_font.mk
--include ./lvgl/src/lv_misc/lv_misc.mk
--include ./lvgl/src/lv_themes/lv_themes.mk
--include ./lvgl/src/lv_draw/lv_draw.mk
--include ./lvgl/src/lv_gpu/lv_gpu.mk
+CSRCS += port/lv_port.c
+CSRCS += port/lv_port_tick.c
 
-CSRCS += lv_tick_interface.c
 
-ifneq ($(CONFIG_USE_LV_FILESYSTEM),)
-CSRCS += lv_fs_interface.c
+ifneq ($(CONFIG_LV_PORT_USE_LCDDEV),)
```

Comment:
why not `ifeq ($(CONFIG_LV_PORT_USE_LCDDEV),y)`?
and the same for other similar places here